examples/profile/controllers/profile.controller.js

```diff
@@ -83,16 +171,21 @@ module.exports = (req, res) => {
     res.render('pages/error', {
       error: 'No token has been provided.',
     });
-    return;
   }
 
   const promise = yotiClient.getActivityDetails(token);
   promise.then((activityDetails) => {
     const profile = activityDetails.getProfile();
+    const attributes = extractAttributesFromProfile(profile);
+    const displayableAttributes = attributes.map(toDisplayableAttribute)
+      .filter((attribute) => !!attribute);
+
+    // DEBUG
+    // console.log('>>>>\n', JSON.stringify(displayableAttributes, null, 2), '\n<<<< displayableAttributes');
```

Comment:
Note: remove this?

Reply:
Yeah, will be, examples are still to be finalised.